src/tests-event-replay/poison-microblock-tests.ts

```diff
@@ -0,0 +1,47 @@
+import { ChainID } from '@stacks/transactions';
+import * as fs from 'fs';
+import { getBnsGenesisBlockFromBlockMessage, getGenesisBlockData } from '../event-replay/helpers';
+import { PgSqlClient } from '../datastore/connection';
+import { getPgClientConfig } from '../datastore/connection-legacy';
+import { databaseHasData, getRawEventRequests } from '../datastore/event-requests';
+import { cycleMigrations, dangerousDropAllTables, runMigrations } from '../datastore/migrations';
+import { PgWriteStore } from '../datastore/pg-write-store';
+import { exportEventsAsTsv, importEventsFromTsv } from '../event-replay/event-replay';
+import { IBD_PRUNABLE_ROUTES, startEventServer } from '../event-stream/event-server';
+import { getIbdBlockHeight, httpPostRequest } from '../helpers';
+import { useWithCleanup } from '../tests/test-helpers';
+
+describe('poison microblock for height 80743', () => {
+  let db: PgWriteStore;
+
+  beforeEach(async () => {
+    process.env.PG_DATABASE = 'postgres';
+    db = await PgWriteStore.connect({
+      usageName: 'tests',
+      withNotifier: false,
+      skipMigrations: true,
+    });
+  });
+
+  afterEach(async () => {
+    await db?.close();
+  });
+
+  test('test that it does not give 500 error', async () => {
```

Comment:
Should this also perform a query to verify that the poison microblock tx was received and is returned? The tsv is pretty opaque and it's not immediately clear to me that it contains the tx type

Reply:
@timstackblock you can query the `txs` table and check the `tx_type` column which will tell you if you have the poison microblock tx ingested 👍

Reply:
Ok thanks for the info I will make the change.